community/about-us.md

```diff
@@ -0,0 +1,36 @@
++++
+toc=true
++++
+
+# About KOSS
+
+* We are a bunch of Open Source enthusiasts, who love to share the culture of OSS. Through events and workshops, we try to share the different aspects of Software Engineering, and onboard folks with the basic understanding and literacy of Software Development.
+  
+* We love to work on new ideas and projects, though we don't promote it from KOSS by default. You can find involvement of KOSS members in MetaKGP, developing on a project which is helpful for the KGP Community. You can also find members who are motivated enough to work on funky ideas and projects, which might range to solve a real-world issue, to create a project for banter, to help out folks learn a new language or framework.
```

Comment:
> involvement of KOSS members in MetaKGP

I am not sure, but this sounds like the "we are everywhere" kind! We can exclude this may be? And the involvement of KOSS in metakgp is a voluntary action. People do it because they like it. Is it necessary to mention it?

NOT a big issue, we can keep this

Reply:
@mukul-mehta @sahil-shubham @thealphadollar   Gymkhana bhi add karna hai??

Reply:
Yup, please do.

Reply:
Done